lametro/management/commands/refresh_guid.py

```diff
@@ -14,39 +17,41 @@ def handle(self, *args, **options):
         scraper.retry_attempts = 0
         scraper.requests_per_minute = 0
 
-        all_topics = scraper.topics()
+        current_topics = set(chain(*Bill.objects.values_list('subject', flat=True)))
```

Comment:
how often are we going to to run this, it seems like this could be pretty slow, and could be moved into SQL.

Reply:
It gets run once a day, and we haven't had any issues so far with execution time. Run locally, it only takes a few seconds.